orc8r/gateway/c/common/datastore/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ list(APPEND PROTO_SRCS "")
 list(APPEND PROTO_HDRS "")
 
 set(ASYNC_ORC8R_CPP_PROTOS common redis)
-set(ASYNC_LTE_CPP_PROTOS session_manager policydb subscriberdb)
+set(ASYNC_LTE_CPP_PROTOS session_manager policydb mobilityd subscriberdb)
```

Comment:
?

Reply:
it fails building as the policydb now references a proto from mobilityd